docs/community_meeting_record.yaml

```diff
@@ -0,0 +1,103 @@
+  - content: Logo discussion
+    proposer: frank-zsy
+    participants: [tedliu1, will-ww]
+  - content: Docker image for sample data
+    proposer: frank-zsy
+    participants: [bifenglin, will-ww]
+  - content: Contribution procedure
+    proposer: frank-zsy
+    participants: []
+
+time: 2020.4.29
+host: sunshinemingo
+attendee: [sunshinemingo, tedliu1, frank-zsy, JingbenShi668, gymgym1212, xiaoya-Esther, wengzhenjie, will-ww, bifenglin, zhuxiangning, birdflyi, yangming]
+agenda:
+  - content: Logo discussion
+    proposer: frank-zsy
+    participants: [tedliu1, will-ww, sunshinemingo, xiaoya-Esther]
+  - content: Pull request collaboration procedure
+    proposer: frank-zsy
+    participants: [hi-rustin]
+  - content: Online global report
+    proposer: frank-zsy
+    participants: [hi-rustin]
+  - content: Credit system details
+    proposer: frank-zsy
+    participants: [gymgym1212]
+  - content: Meeting time
+    proposer: frank-zsy
+    participants: [xiaoya-Esther]
+
+time: 2020.5.13
+host: sunshinemingo
+attendee: [sunshinemingo, tedliu1, frank-zsy, xiaoya-Esther, wengzhenjie, will-ww, bifenglin, zhuxiangning, birdflyi, yangming, dongshu2013, mb0ma, JingbenShi668]
+agenda:
+  - content: Document project name change
+    proposer: frank-zsy
+    participants: []
+  - content: Report visual style
+    proposer: frank-zsy
+    participants: []
+  - content: Logo files added into assets
+    proposer: frank-zsy
+    participants: []
+  - content: Case study of VSCode
+    proposer: frank-zsy
+    participants: [will-ww]
+  - content: Merit system
+    proposer: frank-zsy
+    participants: [will-ww, tedliu1]
+  - content: Summer 2021
+    proposer: sunshinemingo
+    participants: [will-ww, tedliu1]
+  - content: Contributors
+    proposer: tedliu1
+    participants: [will-ww, tedliu1, frank-zsy]
+
+time: 2020.5.20
+host: sunshinemingo
+attendee: [sunshinemingo, xiaoya-Esther, wengzhenjie, will-ww, bifenglin, zhuxiangning, birdflyi, yangming, dongshu2013, JingbenShi668, leofang327, gymgym1212]
+agenda:
+  - content: save as pdf function
+    proposer: frank-zsy
+    participants: [sunshinemingo]
+  - content: Chinese open source projects report
+    proposer: frank-zsy
+    participants: [sunshinemingo]
+  - content: case study issue response & resolve period
+    proposer: frank-zsy
+    participants: [sunshinemingo]
+  - content: OssChat
+    proposer: sunshinemingo
+    participants: [gymgym1212]
+  - content: Summer 2021
+    proposer: sunshinemingo
+    participants: [yangming, leofang327, bifenglin, will-ww]
+
+time: 2020.5.27
+host: sunshinemingo
+attendee: [sunshinemingo, tedliu1, frank-zsy, xiaoya-Esther, wengzhenjie, will-ww, bifenglin, zhuxiangning, birdflyi, gymgym1212, mb0ma]
+agenda:
+  - content: Merit system
+    proposer: sunshinemingo
+    participants: [tedliu1, frank-zsy, gymgym1212, will-ww]
+  - content: Case study requirements
+    proposer: frank-zsy
+    participants: [tedliu1, frank-zsy, xiaoya-Esther, will-ww]
+
```

Comment:
Can you please change file content into a big array instead of records separated by blank lines?